python_modules/dagster-framework/snowflake/dagster_framework/snowflake/solids.py

```diff
@@ -0,0 +1,187 @@
+import sys
+
+from contextlib import closing
+
+import pandas as pd
+import snowflake.connector
+
+import dagster_pandas as dagster_pd
+from dagster import check, Bool, InputDefinition, List, OutputDefinition, Result, SolidDefinition
+
+from .configs import define_snowflake_config
+
+
+class SnowflakeSolidDefinition(SolidDefinition):
+    '''SnowflakeSolidDefinition wraps execution of a list of Snowflake SQL queries.
+
+    Per the Snowflake docs, xecuting multiple SQL statements separated by a semicolon in a single
+    execute call is not supported, and so here we iterate over a list of SQL queries and call
+    connector.execute() on each.
+
+    Attributes:
+
+        name (str): Name of the solid.
+        sql_queries (List[str]): A list of SQL queries to execute together. If auto-commit is
+            enabled, will commit after each query, otherwise will only commit after the last query
+            is completed.
+        parameters (Dict[str, str]): Query parameters to bind to the parameterized query (expects
```

Comment:
maybe make it clear this is one set of parameters for the entire list of queries?

Reply:
yeah, great callout - updated